src/lib/es2020.promise.d.ts

```diff
@@ -0,0 +1,29 @@
+interface PromiseResolvedResult<T> {

```

Comment:
Hi. Drive-by code review here. Shouldn’t this be named something like `PromiseFulfilledResult` to adhere to the [states and fates terminology](https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md)? I think the reason calling it `PromiseResolvedResult` is incorrect here is that while all fulfilled promises are resolved, not all resolved promises are fulfilled. A promise can be resolved to a pending promise, in which case it has not settled, and it can also be resolved to a rejected promise, in which case it has not fulfilled.

If my interpretation of states and fates terminology is correct, shouldn’t the correct name of this interface be `PromiseFulfilledResult`?